src/services/ui/e2e/tests/home/index.spec.ts

```diff
@@ -1,21 +1,37 @@
 import { test, expect } from "@playwright/test";
+import { chromium } from "@playwright/test";
 import * as $ from "@/selectors";
 
+const user = {
+  userName: "george@example.com",
+  password: "bigTUNA1!"
+};
+
```

Comment:
I believe we can remove this object, I don't see it being used

Reply:
yes we def do not want the test pw hardcoded anywhere in the code, even in the git history : (